main.js

```diff
@@ -12,9 +12,42 @@ window.addEventListener('load', trigger.bind(null, 'load'));
 window.addEventListener('load', trigger.bind(null, 'DOMContentLoaded'));
 document.addEventListener('DOMContentLoaded', trigger.bind(null, 'DOMContentLoaded'));
 
+document.onreadystatechange = function () {
+	if (document.readyState === 'complete') {
+		trigger('DOMContentLoaded');
+		trigger('load');
+	}
+};
+
 if (document.readyState === 'complete') {
-	trigger('load');
-	trigger('DOMContentLoaded');
-} else if (document.readyState === 'interactive') {
```

Comment:
Yeah.

This is the source of jQuery for 2.x: https://github.com/jquery/jquery/blob/master/src/core/ready.js
Basically this one uses also only 'complete', probably due to IE9 and 10. It falls back to window.onload.

And the source of jQuery 1.x which supports older IEs as well:
https://github.com/jquery/jquery/blob/1.11-stable/src/core/ready.js